Minor fixes to grammar and capitalization (#188)
hacks-guide/Guide_Wii: docs/faq.md

### How risky is hacking my console?

> Not very risky. The Wii requires much less barrier to entry than other consoles, with some exploits requiring nothing more than a Wii and an internet connection. As long as you follow the general advice laid on the [brick overview page](bricks#brick-prevention), install [BootMii](bootmii) and [Priiloader](priiloader), and use common sense, you should be completely fine!

Reviewer: ```suggestion
If you're hacking your console from a stock state, it is not very risky. The Wii is a much simpler console to hack than others, with some exploits requiring nothing more than a Wii and an Internet connection. As long as you follow the general advice laid out on the [brick overview page](bricks#brick-prevention) and follow the guide steps as listed, you should be fine! If you are particularly worried about making a mistake, feel free to join the Nintendo Homebrew discord at the bottom of this page.
```

This section was due for a rewrite in general, tbh

The Wii can read SD cards of any capacity (even those larger than 32GB), but the card must be [formatted to FAT32/MS-DOS](https://wiki.hacks.guide/wiki/Formatting_an_SD_card) (**not** exFAT or NTFS).

> For USB devices, FAT32/MS-DOS is also recommended. However, users should be aware of FAT32 limitations, which only allow volumes up to 2TB in size and files up to 4GB in size. WBFS was a previously used filesystem type for Wii game backups (not to be confused with WBFS files). Today, it is outdated and should not be used.

Reviewer: ```suggestion
For USB devices, FAT32/MS-DOS is also recommended. However, users should be aware of FAT32 limitations, which only allow volumes up to 2TB in size and files up to 4GB in size. WBFS is filesystem type for Wii game backups (not to be confused with WBFS files) that was commonly used in the past, but today, it is outdated and should not be used.
```

::: danger

SD cards larger than 2TB currently do not exist, any listings for a higher capacity are scams.

:::

> When buying SD cards, it is generally recommended that you stick to well-known brands, especially with the more expensive and larger capacity that you get. You can buy a generic 2GB SD card and most likely it will work fine. However, the same cannot be said for something like a 512GB SD card. General current pricing for SD cards is around 10-35 USD for 256GB and below, whereas 512GB is around 60 USD. When buying off of Amazon, make sure that your card is either `Sold by/Shipped by Amazon` or `Sold by [brand], Shipped by Amazon`. Be careful when buying storage on eBay.

Reviewer: ```suggestion
When buying SD cards, it is generally recommended that you stick to well-known brands, especially with higher capacity cards. You can buy a generic 2GB SD card and most likely it will work fine, however, the same cannot be said for something like a 512GB SD card. General current pricing for SD cards is around 10-35 USD for 256GB and below, whereas 512GB is around 60 USD. When buying off of Amazon, make sure that your card is either `Sold by/Shipped by Amazon` or `Sold by [brand], Shipped by Amazon`. Be careful when buying storage on eBay.
```

For hard drive sizing, 2.5 inch drives should work fine in most cases running off of USB power alone on a Wii, but 3.5 inch drives consume more power and will always need an external power adapter. Make sure to plan for this when determining what you want to do with an external drive.

For Wii U users, it is highly recommended to have a hard drive that takes external power, or to use a powered USB hub and/or Y cable for 2.5 inch drives. This is because the Wii U does not provide enough power in the USB slots as the Wii does.

For Wii mini users, it is recommended to have an externally powered USB hub in order to have external storage, internet adapters, and other peripherals connected simultaneously. You may also want to use a Y cable for drive power.

As for your choices in shopping, this largely comes down to three different scenarios:

* Buying a new external hard drive from scratch: New external drives come brand new around 60 USD for 2TB, 50 USD for 1TB, around 30 USD for sizes lower than 1TB. Try to stick within the three present manufacturing brands listed above, but bear in mind that offerings are sparse for drive sizes lower than 1TB.

> * Buying a new hard drive, and a new enclosure: New hard drives are usually more reliable than buying a secondhand hard drive, and you can buy an enclosure to house this new hard drive. 2.5 inch enclosures are normally powerable off of USB alone, while 3.5 inch enclosures almost always need an external adapter for power. An example price is buying a 500GB Western Digital drive and a 2.5 inch drive enclosure off of Amazon will cost about 30 USD.

Reviewer 2: ```suggestion
* Buying a new hard drive, and a new enclosure: New hard drives are decently cheaper than pre-assembled hard drives, and you can buy an enclosure to house this new hard drive. 2.5 inch enclosures are normally powerable off of USB alone, while 3.5 inch enclosures almost always come with an external adapter for power. An example price is buying a 500GB Western Digital drive and a 2.5 inch drive enclosure off of Amazon, which will cost about 30 USD.
```
